src/repository/SchedulesRepository.ts

```diff
@@ -30,15 +30,15 @@ export class SchedulesRepository extends Repository<ScheduleEntity> {
    * If there is no such schedule -> we try inserting this schedule as the active one.
    * ↳ If it worked return true, otherwise false.
    */
-  async isActiveSchedule(): Promise<boolean> {
+  async isActiveSchedule(name: string): Promise<boolean> {
```

Comment:
I still find it a bit weird that this method which only returns a boolean might also create a brand new schedule for you. But I can't come up with a better name for it or a good way to split the method...

Reply:
Splitting it is not really possible I think, because this needs to be one atomic operation (`findOneAndUpdate`). But yeah, naming is hard here. We can rename it anytime we come up with a better name though, it's internal API.